website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -58,6 +58,17 @@ api.createGroupTasks = {
     let tasks = await createTasks(req, res, {user, group});
 
     res.respond(201, tasks.length === 1 ? tasks[0] : tasks);
+
+    tasks.forEach((task) => {
+      res.analytics.track('create task', {
```

Comment:
7 day age check?

Reply:
Similar to above, I think group plans are uncommon enough at the moment that we don't need to limit it. In fact, restricting to 7 days age would probably exclude the vast majority of group plan task creation activity.